examples/cli/cli_demo.go

```diff
@@ -2196,11 +2198,25 @@ func publishRequest(args []string) {
 		"m2": "n2",
 	}
 
+	payload := map[string]interface{}{
```

Comment:
This doesn't look related to deadlocks. :)

Reply:
This was for the APNS test, I used the example and didn't remove it :(